sites/main-page-title/_base.scss

```
@@ -9,6 +9,16 @@ h1#wb-cont, hgroup#wb-cont h1 {
 	border-top-width: 0;
 }
 
+h1#wb-cont[dir="rtl"], hgroup#wb-cont[dir="rtl"] h1, [dir="rtl"] h1#wb-cont {
```

Missing this selector

```suggestion
h1#wb-cont[dir="rtl"], hgroup#wb-cont[dir="rtl"] h1, [dir="rtl"] h1#wb-cont, [dir="rtl"] hgroup#wb-cont h1 {
```

---

Added. Thanks